geowebcache/wms/src/test/java/org/geowebcache/config/GetCapabilitiesConfigurationTest.java

```diff
@@ -100,6 +105,18 @@ WebMapServer getWMS() {
         assertThat(wmsLayer.getLegends().getLegendsRawInfo().get(0).getWidth(), is(50));
         assertThat(wmsLayer.getLegends().getLegendsRawInfo().get(0).getHeight(), is(100));
         assertThat(wmsLayer.getLegends().getLegendsRawInfo().get(0).getFormat(), is("image/gif"));
+
+        List<ParameterFilter> parameterFilters = wmsLayer.getParameterFilters();
```

Comment:
Clearer and more compact way of doing this

````
assertThat(parameterFilters, containsInAnyOrder(
    hasProperty("key", equalToIgnoringCase("style")), 
    hasProperty("key", equalToIgnoringCase("angle"))));
````

I'd also like to see separate tests for empty and not empty.  The parsing also would also benefit from one, more than one, value with a `&` and value with a `=` and otherwise trying to break it.  Switching to a `Collection` would eliminate that need.